addons/finger/stringtable.xml

```diff
@@ -1,40 +1,50 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project name="ACE">
-    <Package name="finger">
-        <Key ID="STR_ACE_finger_indicatorForSelf_name">
-            <English>Show finger indicator to self</English>
-            <Russian>Отображать пальце-индикатор для показывающего игрока</Russian>
-        </Key>
-        <Key ID="STR_ACE_finger_indicatorForSelf_description">
-            <English>Render the indicator for the pointing player. This option doesn't affect whether the other players would see the indicator</English>
-            <Russian>Отображать индикатор для показывающего игрока. Эта настройка не влияет на то, будутт ли другие игроки видеть индикатор</Russian>
-        </Key>
-        <Key ID="STR_ACE_finger_indicatorColor_name">
-            <English>Finger indicator</English>
-            <Russian>Пальце-индикатор</Russian>
-        </Key>
-        <Key ID="STR_ACE_finger_indicatorColor_description">
-            <English>Color of the finger-pointing indicator circle</English>
-            <Russian>Цвет индикатора пальце-указания</Russian>
-        </Key>
-        <Key ID="STR_ACE_finger_keyComb">
-            <English>Action "point a finger at"</English>
-            <Russian>Действие "показать пальцем на"</Russian>
-        </Key>
-        <Key ID="STR_ACE_finger_keyComb_description">
-            <English>Points, and shows a virtual marker of where you are looking to nearby units.  Can be held down.</English>
-        </Key>
-        <Key ID="STR_ACE_finger_moduleSettings_displayName">
-            <English>Finger Settings</English>
-        </Key>
-        <Key ID="STR_ACE_finger_enabled_displayName">
-            <English>Finger Pointing Enabled</English>
-        </Key>
-        <Key ID="STR_ACE_finger_maxRange_displayName">
-            <English>Finger Max Range</English>
-        </Key>
-        <Key ID="STR_ACE_finger_maxRange_description">
-            <English>How far away players can finger each other. [default: 4]</English>
-        </Key>
-    </Package>
-</Project>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
whole file changed because all stringtables use UNIX endline format and no Windows one, this one has UNIX endline now